core/templates/dev/head/components/answer_group_editor_directive.html

```diff
@@ -82,6 +86,12 @@
     <button type="button" ng-if="isEditable" class="btn btn-default btn-lg oppia-add-rule-button protractor-test-add-answer" ng-click="addNewRule()">
       + Add Another Possible Answer
     </button>
+
+    <div ng-if="isCurrentInteractionTrainable()">
+      <button type="button" class="btn btn-default btn-lg oppia-add-rule-button protractor-test-add-answer oppia-modify-training-data-button" ng-click="openTrainingDataEditor()">
+        <i class="material-icons md-24">playlist_add</i> Modify Training Data
```

Comment:
@prasanna08 I am getting some warnings from my e2e of duplicated "protractor-test-add-answer" button. Is it intended?

Reply:
Hi @hoangviet1993, are you getting something like: Duplicate <class selector> found. Using the first occurence.
If this is the case, then there is no problem IMO.

Reply:
I think the tag should be `protractor-test-modify-training-data-button`. The test was expecting "add-answer" button to not be clickable but the 'Modify training' button is clickable due to incorrect tag.

Reply:
No idea about the tag (that's a question for @prasanna08) but just to note that we shouldn't get the sort of message that @apb7 is describing -- if there are multiple elements with the same class then we should be expecting a list and explicitly using .first(), otherwise we should ensure that the element with that class is unique.

Reply:
@hoangviet1993, no that wasn't intended. I think it is happening bacause I am using same class as the "Add new answer group" button above. Can you file an issue for this? I will fix it asap. Just have to create new class for "Modify training data" button which has same style aa current class.

Reply:
@prasanna08 #5204 is created per your request, PTAL 👍